ios/TabViewImpl.swift

```diff
@@ -73,13 +74,27 @@ struct TabViewImpl: View {
     .onAppear {
```

Comment:
You should change it to `.onChange(of: props.scrollEdgeAppearance ?? "") {}` to allow for dynamic changing currently users can set this value only on first mount.